docs/usage.md

```diff
@@ -77,4 +81,11 @@ provider "myprovider" {}
 resource "myprovider_myresource" "myresource" {
   # ...
 }
+
+# Using provider functions
+output "example" {
+  value = provider::myprovider::myfunction("input")
+}
```

Comment:
Can you break these four lines out into it's triple-tick own block? That way we can keep the hiliting for the stuff above it.